tests/acceptance/test_performance_summary.py

```diff
@@ -85,6 +85,29 @@ def test_view_details_from_summary(self, mock_now):
             self.page.wait_until_loaded()
             self.browser.snapshot("performance event details")
 
+    @patch("django.utils.timezone.now")
+    def test_tags_page(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+
+        tags_path = "/organizations/{}/performance/summary/tags/?{}".format(
+            self.org.slug,
+            urlencode({"transaction": "/country_by_code/", "project": self.project.id}),
+        )
+
+        # Create a transaction
+        event_data = load_data("transaction", timestamp=before_now(minutes=3))
+        # only frontend pageload transactions can be shown on the vitals tab
+        event_data["contexts"]["trace"]["op"] = "pageload"
+        event_data["measurements"]["fp"]["value"] = 5000
```

Comment:
Is this required for the tags tab? And do we want to mock some data for the heatmap?

Reply:
Removed the munging since it wasn't need. As for the heatmap, we don't visually test echarts.